src/config.py

```diff
@@ -14,5 +19,12 @@ def update_config():
   with open('config/2019.json', 'r') as config_file:
     config_json['2019'] = json.load(config_file)
 
-
+    for contributor_id, contributor in config_json['2019']['contributors'].items():
+      if 'gravatar' in contributor:
+        gravatar_url = "https://www.gravatar.com/avatar/" + hashlib.md5(contributor['gravatar'].lower().encode()).hexdigest() + ".jpg?"
```

Comment:
Nit: Mixed use of single and double quotes, prefer to use single quotes throughout.